backend/requirements-dev.in

```diff
@@ -1,5 +1,5 @@
 -c requirements.txt
-black
+git+https://github.com/psf/black
```

Comment:
This is really cool.  I didn't know you could specify a git repo directly in the requirements.txt file.